scripts/ci/docker-compose/backend-none.yml

```diff
@@ -15,19 +14,11 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
```

Comment:
This one caused that partial lines of output were not visible when tests were running until EOL, got rid of it by vendoring in "capture warnings" plugin and modifying it so that we do not have to filter the warnings  and see "real" test output at the end of test session.